src/shared/components/chip/chip-info/chip-info-item.tsx

```diff
@@ -0,0 +1,18 @@
+import type { FC } from 'react';
+import Icon from '@components/Icon';
+
+interface ChipInfoItemProps {
+  iconName: string;
+  text: string;
+}
+
+const ChipInfoItem: FC<ChipInfoItemProps> = ({ iconName, text }) => {
+  return (
+    <div className="flex items-center text-gray-600 gap-[0.4rem] pr-[0.8rem] py-[0.4rem] rounded-[4px]">
```

Comment:
```suggestion
    className={cn(
  'flex items-center gap-[0.4rem] pr-[0.8rem] py-[0.4rem] rounded-[4px]',
  className
)}
```

이런식으로 cn함수 적용하는게 더 깔끔할 것 같기도 합니다!!